### PR TITLE
Api tweaks

### PR DESCRIPTION
**Pull request by Rikuoja**

### CODE REVIEW

**Comment by mikkokeskinen** [comment]
I think HyperlinkedRelatedField is not the best experience for an API consumer. The hyperlinks are in my opinion only convenient for a casual browser.

It should be the html-views responsibility to augment the data with a usable link when it's appropriate.

File: hours/tests/test_translation_serializer.py [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+from django.conf import settings
+from pytz import timezone
 
 from hours.models import Resource
 from hours.serializers import ResourceSerializer
 
 
 @pytest.mark.django_db
 def test_to_representation(resource):
+    now = datetime.datetime.now(timezone(settings.TIME_ZONE))
```
Comment (mikkokeskinen) [inline hours/tests/test_translation_serializer.py:13]: Could this be `django.utils.timezone.now()`?

**Review by mikkokeskinen — CHANGES_REQUESTED** [review]
We should think about the hyperlinks.

File: hours/tests/test_translation_serializer.py (hunk 1 above)
Comment (Rikuoja) [reply hours/tests/test_translation_serializer.py]: Good point, also changed the serializer to return UTC times.

**Pull request closed without merging**